cleanlab/experimental/tf_nlp.py

```diff
@@ -0,0 +1,74 @@
+"""
+A Scikeras classifier (adapted for huggingface models) which
+can be used for finding label issues in any nlp dataset.
+"""
+
+from scikeras.wrappers import KerasClassifier
+from sklearn.preprocessing import FunctionTransformer
+import tensorflow as tf
+import numpy as np
+
+
+class MultiInputKerasClassifier(KerasClassifier):
```

Comment:
Can you propose a more precise name for this class?  It doesn't appear to me to be a generic MultiInputKerasClassifier. Eg. the inputs are expected to be text and of a very particular format with `train_input` having keys with specific names like ""input_ids", "attention_mask",